tests/behat/behat-features/replication-aditions.feature

```diff
@@ -0,0 +1,208 @@
+@api
+@replication @replication-aditions
+Feature: Contentpool media replication basically works.
+
+  Make sure that media referenced in content can be pulled from remote contentpool server.
+
+  Scenario:
+    Given I am logged in to contentpool
+
+    # Add media entities
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Media image 1"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Media image 2"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+
+    And I visit path "media/add/image" on contentpool
+    And I fill in "name[0][value]" with "BEHAT: Media image 3"
+    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]"
+    And I press "Save and publish"
+
+  @javascript @replication-aditions-default
+  Scenario: Media attached to replicated entities is replicated
+    # Create article with teaser
+    Given I am logged in to contentpool
+    And I visit path "node/add/article" on contentpool
+    And I fill in "title[0][value]" with "BEHAT: Media article"
+    And I fill in "field_seo_title[0][value]" with "Behat media article"
+    And I select "-Cooking" from "edit-field-channel"
+    And I press "field_teaser_media_entity_browser_entity_browser"
+    Then I wait for AJAX to finish
+    When I click on media "BEHAT: Media image 1" in entity browser "image_browser"
+    And I click on "#edit-submit" in entity browser "image_browser"
+    And I wait for entity browser "image_browser" to close
+    Then I wait for AJAX to finish
+    And I select "Published" from "moderation_state[0]"
+    And I press "Save as"
+    And I wait for the page to be loaded
+
+    # Replicate
+    Given I open the satellite
+    When I run drush cppull
+    Then drush output should contain "Content of remote /Contentpool/ has been replicated successfully."
+    And I am logged in as a user with the "administrator" role
+
+    # Check article replicated
+    And I am on "admin/content"
+    Then I should see the text "BEHAT: Media article"
+
+    # Check media is replicated
+    And I am on "admin/content/media"
+    Then I should see the text "BEHAT: Media image 1"
```

Comment:
please also validate only used media is validated.
-> BEHAT: Media image 2 should not be validated here.

Reply:
update: seems to be covered below. so can stay without also.